rsearch/config.go

```diff
@@ -0,0 +1,48 @@
+package rsearch
+
+import (
+	"gopkg.in/gcfg.v1"
+)
+
+// Done is an alias for empty struct, used for termination channels
```

Comment:
It's not a good habit to alias the types if you're not adding methods.
`struct{}` is a fairly ugly type, to be fair.